pkg/client/connector/install.go

```diff
@@ -94,14 +94,18 @@ func (ki *installer) createManagerSvc(c context.Context) (*kates.Service, error)
 	}
 
 	// Ensure that the managerNamespace exists
-	if !ki.namespaceExists(managerNamespace) {
+	_, err := ki.findNamespace(c, managerNamespace)
+	if err != nil {
 		ns := &kates.Namespace{
```

Comment:
Should this proceed for other errors than `errors2.IsNotFound(err)` ?

Reply:
Ah good point, fixed!